docs/eso/eso.rst

```diff
@@ -284,37 +302,40 @@ Only a small subset of the keywords presents in the data products can be obtaine
 with :meth:`~astroquery.eso.EsoClass.query_instrument` or :meth:`~astroquery.eso.EsoClass.query_main`.
 There is however a way to get the full primary header of the FITS data products,
 using :meth:`~astroquery.eso.EsoClass.get_headers`.
-This method is detailed in the example below, continuing with the previously obtained table.
+This method is detailed in the example below.
+
 
-.. code-block:: python
+.. doctest-remote-data::
 
+    >>> table = eso.query_instrument('midi', column_filters={'target':'NGC 4151',
+    ...                                                      'stime':'2007-01-01',
+    ...                                                      'etime':'2008-01-01'},
+    ...                              columns=['night'])
     >>> table_headers = eso.get_headers(table['DP.ID'])
-    >>> table_headers.pprint()
-                 ARCFILE              BITPIX ...   TELESCOP     UTC
-    --------------------------------- ------ ... ------------ -------
-    MIDI.2007-02-07T07:01:51.000.fits     16 ... ESO-VLTI-U23 25300.5
-    MIDI.2007-02-07T07:02:49.000.fits     16 ... ESO-VLTI-U23 25358.5
-    MIDI.2007-02-07T07:03:30.695.fits     16 ... ESO-VLTI-U23 25358.5
-    MIDI.2007-02-07T07:05:47.000.fits     16 ... ESO-VLTI-U23 25538.5
-    MIDI.2007-02-07T07:06:28.695.fits     16 ... ESO-VLTI-U23 25538.5
-    MIDI.2007-02-07T07:09:03.000.fits     16 ... ESO-VLTI-U23 25732.5
-    MIDI.2007-02-07T07:09:44.695.fits     16 ... ESO-VLTI-U23 25732.5
-    MIDI.2007-02-07T07:13:09.000.fits     16 ... ESO-VLTI-U23 25978.5
-    MIDI.2007-02-07T07:13:50.695.fits     16 ... ESO-VLTI-U23 25978.5
-    MIDI.2007-02-07T07:15:55.000.fits     16 ... ESO-VLTI-U23 26144.5
-    MIDI.2007-02-07T07:16:36.694.fits     16 ... ESO-VLTI-U23 26144.5
-                                  ...    ... ...          ...     ...
-    MIDI.2007-02-07T07:51:13.485.fits     16 ... ESO-VLTI-U23 28190.5
-    MIDI.2007-02-07T07:52:27.992.fits     16 ... ESO-VLTI-U23 28190.5
-    MIDI.2007-02-07T07:56:21.000.fits     16 ... ESO-VLTI-U23 28572.5
-    MIDI.2007-02-07T07:57:35.485.fits     16 ... ESO-VLTI-U23 28572.5
-    MIDI.2007-02-07T07:59:46.000.fits     16 ... ESO-VLTI-U23 28778.5
-    MIDI.2007-02-07T08:01:00.486.fits     16 ... ESO-VLTI-U23 28778.5
-    MIDI.2007-02-07T08:03:42.000.fits     16 ... ESO-VLTI-U23 29014.5
-    MIDI.2007-02-07T08:04:56.506.fits     16 ... ESO-VLTI-U23 29014.5
-    MIDI.2007-02-07T08:06:11.013.fits     16 ... ESO-VLTI-U23 29014.5
-    MIDI.2007-02-07T08:08:19.000.fits     16 ... ESO-VLTI-U23 29288.5
-    MIDI.2007-02-07T08:09:33.506.fits     16 ... ESO-VLTI-U23 29288.5
+    >>> table_headers.pprint()  # doctest: +SKIP
```

Comment:
Do IGNORE_OUTPUT rather than SKIP for these cases, to make sure we still catch if there is any failure